### DISCUSSION TITLE
Support VIM j,k,h,l fast move cursor

### DISCUSSION THREAD

**Discussion by JerryZheng89** [isontheline/pro.webssh.net#1114] [Ideas]
When i use vim in webssh, long press (j,k,h,l) keys button, it cannot move fast toward down, up, left, and right, and the cursor don't move.

**Comment by isontheline** [comment]
Hello @JerryZheng89 👋 

An issue is already opened here : https://github.com/isontheline/pro.webssh.net/issues/976

Yet WebSSH lacks of key repeat.

I'll try to implement this feature
